src/main/groovy/me/tatarka/RetrolambdaPluginAndroid.groovy

```diff
@@ -123,7 +123,8 @@ public class RetrolambdaPluginAndroid implements Plugin<Project> {
         }
 
         JavaCompile compileUnitTest = (JavaCompile) project.tasks.find { task -> 
-            task.name.startsWith("compile${variant.capitalize()}UnitTestJava") 
+            task.name.startsWith("compile${variant.capitalize()}UnitTestJava") ||
+            task.name.startsWith("compile${variant.capitalize()}UnitTestJavaWithJavac")
```

Comment:
How does `startsWith("compile${variant.capitalize()}UnitTestJava")` not match "compileFooUnitTestWithJavac"?